src/main/java/org/broadinstitute/hellbender/tools/copynumber/utils/annotatedregion/SimpleAnnotatedGenomicRegion.java

```diff
@@ -59,6 +56,34 @@ public int getEnd() {
         return interval.getEnd();
     }
 
+    public void setInterval(final Locatable interval) {
+        this.interval = new SimpleInterval(interval);
+    }
+
+    public void setEnd(final int end) {
```

Comment:
Again, it seems unnecessary to have setters and make this mutable.  Also, `setInterval` is not actually used anywhere and `setAnnotation` unnecessarily returns a `String`.

To give you an idea of how idiosyncratic this is: If we merged this code, these would actually be the only setter methods in the entire CNV codebase!

Reply:
I'm not even sure that I need the setters.  This might be forgotten leftovers.  I'll delete.

Reply:
The setters are deleted.  `setAnnotation()` no longer returns a string.

Not quite done yet.